package.json

```diff
@@ -117,36 +118,59 @@
 			}
 		}
 	},
+	"npmPackageJsonLintConfig": {
+		"extends": "@wordpress/npm-package-json-lint-config",
+		"rules": {
+			"require-publishConfig": "error",
+			"valid-values-author": [
+				"error",
+				[
+					"WordPress"
+				]
+			],
+			"valid-values-publishConfig": [
+				"error",
+				[
+					{
+						"access": "public"
+					}
+				]
+			]
+		}
+	},
 	"scripts": {
-		"prebuild": "check-node-version --package",
+		"prebuild": "npm run check-engines",
 		"build:packages": "rimraf ./packages/*/build ./packages/*/build-module && node ./bin/packages/build.js",
 		"build": "npm run build:packages && cross-env NODE_ENV=production webpack",
-		"lint": "eslint .",
-		"lint:fix": "eslint . --fix",
-		"lint-php": "docker-compose run --rm composer run-script lint",
-		"predev": "check-node-version --package",
+		"check-engines": "check-node-version --package",
+		"ci": "concurrently \"npm run lint && npm run build\" \"npm run test-unit:coverage-ci\"",
```

Comment:
Minor: `npm-run-all` could be used here now in place of `npm run lint && npm run build`. I think they could occur in parallel as well? **Edit:** Though, the fact we're using `concurrently` separate here makes me think we needed them to occur in sequence. Sounds familiar, hmm..

Reply:
I spend some time trying to rewrite it with `npm-run-all` but failed, so I removed this dependency at all in favor of `concurrently` :)